### DISCUSSION TITLE
Insight Panel Selection should use the same UX as Field Setup

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#16539] [Feature Requests]
We should reuse the Add Field design for how Insight Panels (and perhaps Flow Operations) are chosen.

![CleanShot 2022-11-20 at 16 00 45](https://user-images.githubusercontent.com/522079/202925889-6ac8806b-b186-4b4a-b99b-f12572074e32.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Reply by azrikahar** [reply]
I believe this is actually an accepted request as there's an open issue for this in #17061. Moving this into Approved Requests for now unless we'd like to close that issue for the time being 👍

**Comment by rijkvanzanten** [comment]
This feature request has been accepted and is queued to be implemented! You can follow along with the progress here:https://github.com/directus/directus/issues/17061
